src/main/target/system_stm32f4xx.c

```diff
@@ -540,7 +540,17 @@ void SystemInit(void)
 
   /* Configure the Vector Table location add offset address ------------------*/
 #ifdef VECT_TAB_SRAM
-  SCB->VTOR = SRAM_BASE | VECT_TAB_OFFSET; /* Vector Table Relocation in Internal SRAM */
+  extern uint8_t isr_vector_table_flash_base;
+  extern uint8_t isr_vector_table_base;
+  extern uint8_t isr_vector_table_end;
```

Comment:
Can't those be ```uint32_t```? You're recasting them later anyway.

Reply:
They could be. I always choose the smallest addressable object because these symbols are non-attributed addresses. If I could declare them `void`, then I would ;)

Reply:
one possibility is to declare this as `extern uint32_t isr_vector_table_flash_base[1]`, getting address directly. But simple uint8_t is probably a bit better.